ui/packages/consul-ui/app/components/topology-metrics/stats/index.hbs

```diff
@@ -15,7 +15,7 @@
   />
 {{/if}}
 
-<div class="stats">
+<div class="topology-metrics-stats" data-test-topology-metrics-stats>
```

Comment:
Could we splat an attributes on here also while you are here? Sorry I should have mentioned before, it's only just come to mind.

Reply:
@johncowen what attributes?

Reply:
Oh sorry, `...attributes`, they are informally known as splattributes

See the bottom of this docs page:

> The ...attributes syntax determines where the attributes from a tag should appear in the component's template. Any number of attributes can be specified on the avatar component now, and they will all end up on the element that has ...attributes.

> In general, you should place ...attributes after any attributes you specify to give people using your component an opportunity to override your attribute.

https://guides.emberjs.com/release/components/component-arguments-and-html-attributes/

Reply:
👍